go.mod

```diff
@@ -1,8 +1,14 @@
 module github.com/percona/promconfig
 
-go 1.14
+go 1.25.3
```

Comment:
This project used widely by community. Forcing it to use such version may be a blocker for some users. Not sure that it worth to make this change, especially if we don't need any features from latest versions.

Reply:
I checked what the community uses and discovered that it's a pretty [recent version](https://github.com/prometheus/prometheus/blob/main/go.mod#L3) of Go, even though it has known CVEs.